package.json

```diff
@@ -99,6 +103,8 @@
     "@antfu/eslint-config": "^0.33.1",
     "@babel/parser": "^7.20.5",
     "@babel/types": "^7.20.5",
+    "@nuxt/kit": "^3.0.0",
```

Comment:
We should put it to dependencies instead of dev deps

Reply:
done